website/src/pages/subscriptions/create.astro

```diff
@@ -18,150 +18,182 @@ const variantFilter = {
-                            </li>
-                            <li>
-                                <div class='join flex items-center'>
-                                    <label class='join-item'>
-                                        <input class='input join-item input-bordered' placeholder='test2@test.test' />
-                                    </label>
-                                    <button class='btn join-item'>
-                                        <div class='iconify mdi--trash-can-outline'></div>
-                                    </button>
-                                </div>
-                            </li>
-                            <li>
-                                <button class='btn btn-ghost btn-sm text-gray-400'>Add</button>
-                            </li>
-                        </ul>
-                    </li>
-                    <li>
-                        <label class='flex gap-2'>
-                            <input type='checkbox' class='checkbox' />
-                            Slack
-                        </label>
-                        <div class='ml-8 mt-2 flex flex-col'>
-                            <label>
-                                <input class='input input-bordered' placeholder='Channel info' />
-                            </label>
-                        </div>
-                    </li>
-                </ul>
+                <div>
+                    <h3 class='mb-2'>Name</h3>
+                    <p class='mb-2 max-w-xl text-sm text-gray-400'>
+                        A name for the subscription. This will be used to identify the subscription in the list of
+                        subscriptions.
+                    </p>
+                    <label class='w-full'>
+                        <input
+                            class='input input-sm input-bordered w-full max-w-xl'
+                            placeholder='Your name for the subscription'
+                        />
+                    </label>
+                </div>
+
+                <div>
+                    <h3 class='mb-2'>Interval</h3>
+                    <p class='mb-2 max-w-xl text-sm text-gray-400'>
+                        How often you want to receive notifications. You can choose between daily, weekly, and monthly.
+                    </p>
+                    <label class='w-full'>
+                        <select class='select select-bordered select-sm w-full max-w-xl'>
+                            <option>daily</option>
+                            <option>weekly</option>
+                            <option>monthly</option>
+                        </select>
+                    </label>
+                </div>
+
+                <div>
+                    <h3 class='mb-2'>Pathogen</h3>
+                    <p class='mb-2 max-w-xl text-sm text-gray-400'>
+                        The pathogen you want to receive notifications for. You can choose between COVID-19 and Flu.
```

Comment:
Same here. The list will have to be made dynamic at some point and this description is likely to be forgotten.
```suggestion
                        The pathogen you want to receive notifications for.
```